### DISCUSSION TITLE
keyboard shortcuts, plugin

### DISCUSSION THREAD

**Discussion by yo2ldk** [AlexandreRouma/SDRPlusPlus#716] [Q&A] [answered]
Hi,
  
  it is possible to ad more keyboard shortcuts,please ?
  like zoom in out, audio zoom, mode, bad, etc?
  freq up down work well
    Thank you and my best 73 !

**Comment by the-gobleo-tinkerer** [comment, marked as answer]
By "audio zoom" did you mean audio volume (loudness)? And what did you mean by "bad"?

**Reply by yo2ldk** [reply]
"bad"  = band, my keyboard not take it all
"audio zoom"  my mistake , i think it at sdr# who have also audio spectrum;
but yes, can be volume up down..

**Comment by kaushleshchandel** [comment]
Hi, I am also looking for something similar. I am building SDR Dock, with VFO knobs, and sliders to control SDR. I wish there were keyboard shortcuts for stuff like Frequency, Squelch, Mode, Gain, Spectrum zoom, etc. That way I can program the custom controls with keyboard shortcuts.

 [https://hackaday.io/project/192534-sdrdock](url)

![PISDR](https://github.com/AlexandreRouma/SDRPlusPlus/assets/10075547/29710d0d-d97b-40a4-947d-0881564a3f4e)
